nise/generators/ocp/ocp_generator.py

```diff
@@ -88,20 +89,58 @@
     "namespace",
     "namespace_labels",
 )
+OCP_ROS_USAGE_COLUMN = (
+    "interval_start",
+    "interval_end",
+    "report_period_start",
+    "report_period_end",
+    "namespace",
+    "node",
+    "resource_id",
+    "pod",
+    "container_name",
+    "deployment_name",
```

Comment:
Sorry to have you change this again, but let's remove `deployment_name` and replace it with `owner_name,owner_kind,workload,workload_type`.

owner_kind is one of `StatefulSet, ReplicaSet, ReplicationController, DaemonSet, Job` which correspond to workload_type which are one of `statefulset, deployment, deploymentconfig, daemonset, job`. So when `workload_type = deployment`, then `owner_kind = ReplicaSet`, and `workload_type = deploymentconfig`, then `owner_kind = ReplicationController` etc.